src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -395,8 +395,10 @@ public function prepend(ContainerBuilder $container): void
     /**
      * Checks if the classification of media is enabled.
      *
-     * @param array<string, class-string> $bundles
-     * @param array<string, mixed>        $config
+     * @param array<string, string> $bundles
+     * @param array<string, mixed>  $config
+     *
+     * @phpstan-param array<string, class-string> $bundles
```

Comment:
For the record, class-string is now suported by PHPStorm.

Don't know if it's worth moving it to `@phpstan-param` then.

Reply:
I guess nothing it is just an alias right?

Reply:
I assume you were talking bout `array-key`.

Just ask on slack and
- `array<array-key, Foo>` is the same than `array<Foo>` 
- array-key is for int|string what mixed is for int|string|float|bool|... ; best would be to precise the keys.

In the case of
https://github.com/sonata-project/SonataMediaBundle/pull/2017/files#diff-ff8944e4b35328a08ac08926ea1feb404a7261265fa7bbd12fa923154613589dR191
looking at getFormatNamesByContext
https://github.com/sonata-project/SonataMediaBundle/blob/master/src/Provider/Pool.php#L181
we can precise the signature to `array<string, string>`.

In the case of 
https://github.com/sonata-project/SonataMediaBundle/pull/2017/files#diff-280f9f70126ae1825f3748a0096963b6d600ebc5ca38b8ffe532214ebc654eaaR252
the array-key of the collection https://github.com/sonata-project/SonataMediaBundle/blob/master/src/Model/Gallery.php#L53 could be changed to `int`
and so on

Reply:
Fixed all of them